nginx.conf

```
@@ -79,3 +79,83 @@ server{
         try_files $uri $uri/ $uri/index.html =404;
     }
 }
+
+
+# serve the tech-fair site
+server{
+    server_name tech-fair.sfucsss.org;
+    listen 80;
+
+    # 
+    root /var/www/html/tech-fair-sfucsss;
+
+    access_log /var/www/logs/csss-site-backend/tech-fair-nginx-access.log;
+    error_log /var/www/logs/csss-site-backend/tech-fair-nginx-error.log;
+
+    # proxy csss-site-backend
+    location /api/ {
+        rewrite ^/api/(.*)$ /$1 break;
+
+        keepalive_timeout 5;
+        client_max_body_size 1G; # Was 4G
+
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_pass http://backend;
+
+        # update cors header to allow requests from test site
+        add_header Access-Control-Allow-Origin https://tech-fair.sfucsss.org always;
+        add_header Access-Control-Allow-Credentials true;
+    }
+
+    # redirects old 2024 mountain madness requests to the new URL
+    location ~ ^/events/2024/mm(/|/index.html)?$ {
+	    return 301 /mountain_madness/2024/index.html;
+    }
+
+    location / {
+        charset utf-8;
+        try_files $uri $uri/ $uri/index.html =404;
+    }
+}
+# serves the frosh site
+server{
+    server_name frosh.sfucsss.org;
+    listen 80;
+
+    # 
+    root /var/www/html/frosh-sfucsss;
```

```suggestion
    root /var/www/html/frosh;
```
 To match the folder structure I'll use